collectors/python.d.plugin/alarms/README.md

```diff
@@ -0,0 +1,46 @@
+<!--
+title: "Alarms"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/alarms/README.md
+-->
+
+# Alarms - graphing active Netdata alarms over time
+
+This collector creates an 'Alarms' section with one line plot showing alarms by status. Alarm states are mapped to integer values according to the below mapping.
+
+```
+{'UNDEFINED': 0, 'CLEAR': 0, 'WARNING': 1, 'CRITICAL': 2}
+```
```

Comment:
- Why do we use same value (`0`) for different statuses - 'UNDEFINED' and 'CLEAR'?

- [According docs](https://learn.netdata.cloud/docs/agent/health/reference#alarm-statuses) there is more then 4 statuses, do we need only last 4?:

> $REMOVED, $UNINITIALIZED, $UNDEFINED, $CLEAR, $WARNING, $CRITICAL

Reply:
it's not really clear to me how to map all 5 statuses to numbers that have sort sort of order. 

it's obvious that WARNING < CRITICAL so makes sense for 1 < 2 but not sure what numbers we would use for the other status. 

I guess it also makes sense in this setting that CLEAR = 0. 

So maybe question is what to do with UNDEFINED, UNINITIALIZED, REMOVED. SO which of these would you care to visualize. Maybe you might want to see UNDEFINED as it means alarm failed to calculate - so maybe UNDEFINED = -1 perhaps. 

Thoughts?

Not sure I know enough about the different alarm statuses to be sure about this.

Reply:
Well, if this collector collects and visualises **alarms statuses**, then we need to visualise them all, so we have history of statuses changes 🤔 

> but not sure what numbers we would use for the other status.

Just enum - 6 statuses, 0-5 (or 1-6).

Reply:
If this collectors suppose to show only `warn` and `crit` statuses, then:
- `warn`: 1
- `crit`: 2
- any other: 0

Reply:
If all (looks better to me) => `clear`: 0, all other 1-5.

Reply:
I think you only really want to see `warn` or `crit` so you can look at it and all 0 means no warning or critical alarms, likewise as you scrolling back in time. 

Having to look at the chart and remember in your head what a value of 3 or 4 means could be confusing I think. 

Maybe its just about re-wording things or perhaps could be something we could make configurable. 

Would be easy enough to make configurable and then just need to decide the default which is maybe an easier decision - thoughts? I can have a go at making it configurable if we want.